aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: Learn about the new features in ASP.NET Core 7.0.
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/09/2022
+ms.date: 08/15/2022
```

Comment:
Thanks for all the clean up here. BTW, we like to do two PR's, first on the topic where you have `contributes to #1234` and after that merges a faster one to what's new, just so we don't tie up what's new.

Reply:
Got it, thanks.